ddtrace/llmobs/_llmobs.py

```diff
@@ -230,16 +230,23 @@ def disable(cls) -> None:
         log.debug("%s disabled", cls.__name__)
 
     @classmethod
-    def annotation_context(cls, tags: Optional[Dict[str, Any]] = None) -> AnnotationContext:
+    def annotation_context(
+        cls, tags: Optional[Dict[str, Any]] = None, prompt: Optional[dict] = None, name: Optional[str] = None
+    ) -> AnnotationContext:
         """
         Sets specified attributes on all LLMObs spans created while the returned AnnotationContext is active.
-        Do not use nested annotation contexts to override the same tags since the order in which annotations
+        Do not use nested annotation contexts to override the same attributes since the order in which annotations
         are applied is non-deterministic.
 
         :param tags: Dictionary of JSON serializable key-value tag pairs to set or update on the LLMObs span
                      regarding the span's context.
+        :param prompt: A dictionary that represents the prompt used for an LLM call in the following form:
+                    {"template": "...", "id": "...", "version": "...", "variables": {"variable_1": "value_1", ...}}
```

Comment:
Would it make sense to use a dataclass, typed dict, or something that specified the required/optional fields?

Reply:
Thanks for the suggestion - I added a note about the `ddtrace.llmobs.utils.Prompt` `TypedDict` class